roles/marathon/defaults/main.yml

```diff
@@ -21,6 +21,8 @@ marathon_keystore_password: ""
 marathon_port: 18080
 marathon_proxy_port: 8080
 
+marathon_log_level: warn
```

Comment:
Might also be good to put this in the Marathon README, or at least put an inline comment with valid options